pkg/buses/message_dispatcher_test.go

```diff
@@ -110,7 +111,7 @@ func TestDispatchMessage(t *testing.T) {
 					"knative-2":      "knative-2-value",
 					"ce-abc":         "ce-abc-value",
 				},
-				Payload: []byte("replyTo"),
+				Payload: []byte("reply"),
```

Comment:
This might be unintentional as well.

Reply:
This one is debatable, but probably correct. Either way, it was changed consistently.